### ISSUE TITLE
Small acrosomal area

### ISSUE THREAD

**Issue by MagoWyr** [obophenotype/human-phenotype-ontology#9841]
**Preferred term label:**
Small acrosomal area

**Synonyms**

**Definition (free text, please give PubMed ID)**
For illustration see: WHO laboratory manual for the examination and processing of human semen (6th Edition)

**Parent term (use hpo.jax.org/app)**
HP:0012865

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA: 399808

**Your nano-attribution (ORCID)**
0000-0002-7651-2403

**Comment by pnrobinson** [comment]
@MagoWyr 
adding as
Small acrosomal area
HP:0034810
def: Reduction in size of the acrosome below  the lower limit of normal. The acrosome is a unique membranous organelle located over the anterior part of the sperm nucleus. According to the World Health Organization criteria, the acrosome must be clearly visible, comprising approximately 40-70 per cent of the sperm head
PMID:31939391

**Comment by MagoWyr** [comment]
Yes, that is correct!
________________________________
